src/status_im/common/home/actions/view.cljs

```diff
@@ -456,11 +465,17 @@
   (let [current-pub-key (rf/sub [:multiaccount/public-key])]
     [quo/action-drawer
      [[(view-profile-entry public-key)
+       (when-not (= current-pub-key public-key)
+         (when config/show-not-implemented-features?
+           (rename-entry)))
+       (when config/show-not-implemented-features?
+        (show-qr-entry))
+       (when config/show-not-implemented-features?
+         (share-profile-entry))]
```

Comment:
I wrapped these not-implemented features individually because it seemed to simpler than refactoring this expression into a conditional merge of items. And it might be easier to work on each not-implemented feature later. Thoughts?